reporting-engine-worker/src/main/java/org/ihtsdo/termserver/scripting/reports/release/PackageComparisonReport.java

```diff
@@ -45,8 +66,11 @@ public void postInit() throws TermServerScriptException {
 	@Override
 	public Job getJob() {
 		JobParameters params = new JobParameters()
-				.add(A_RELEASE).withType(JobParameter.Type.STRING)
-				.add(B_RELEASE).withType(JobParameter.Type.STRING)
+				.add(PREVIOUS_RELEASE).withType(JobParameter.Type.STRING)
+				.add(PREVIOUS_RELEASE_ZIP).withType(JobParameter.Type.STRING)
```

Comment:
I think you want to make these mandatory?   Use .withMandatory()

Reply:
@pgwilliams old code, but will add mandatory where applicable. Thanks.